timed/authentication.py

```diff
@@ -12,13 +12,10 @@
 
 
 class TimedOIDCAuthenticationBackend(OIDCAuthenticationBackend):
-    def get_introspection(self, access_token, id_token, payload):
+    def get_introspection(self, access_token, _id_token, _payload):
```

Comment:
Just a note (no change required): This naming pattern really only works when the method is called with positional args. We'll have to be careful to check the call sites in such cases - especially if it's implementing a protocol / defined API or overriding a base class' implementation